### DISCUSSION TITLE
Does Anyone have any exploits kinda like goguardian or securly but for netsupport?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1126] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I have been looking for a exploit that works with netsupport dna agent and school student but no exploit for like securly or goguardian work for netsupport

**Comment by S-PScripts** [comment]
i think some one had one on the titanium network discord

**Reply by Balistic123** [reply]
Could you link to the comment on the discord server?

**Reply by S-PScripts** [reply]
search for it, its somewhere in the kajigs channel and i believe it has netsupport in its name
